Fix `snakeCaseToCamelCase` for numbers

tnull opened the pull request:
Fixed #96

tnull committed:
Fix `snakeCaseToCamelCase` for numbers

arik-so reviewed (changes requested):
Would you mind adding a couple generator tests for this?

tnull committed:
Fix generator test cases

tnull committed:
Add `FeeEstimator` test case

tnull commented:
> Would you mind adding a couple generator tests for this?

Now fixed the test cases that were failing in `HEAD`, and added a test case for `FeeEstimator`, covering `getEstSatPer1000Weight`.

arik-so commented:
A bunch of the JS tests are unfortunately still failing for me, but that's because we don't have the lightning_01.h file available to them. That file is available here: https://github.com/arik-so/ldk-parser/blob/main/res/lightning_01.h

Let's copy it to `./test/fixtures/lightning_01.h` and edit all the generator and parser tests to refer to it from that location. It might also be helpful to add an `npm test` step to a Github action, might as well do it in here to let CI make sure tests pass.

I apologize for the scope creep.

tnull committed:
Add `lightning_01.h` fixture

arik-so commented on test/fixtures/lightning_01.h:
```diff
@@ -0,0 +1 @@
+../../ldk-c-bindings/lightning-c-bindings/include/lightning.h
```
wait, no, this looks like a symlink. It should be the actual file

tnull replied:
Yes, already fixed. Pushed before realizing my error.

tnull committed:
Add `npm test` to CI
